src/Web/HTML/Window/AnimationFrame.purs

```diff
@@ -0,0 +1,10 @@
+module Web.HTML.Window.AnimationFrame where
+
+import Prelude
+
+import Effect (Effect)
+import Web.HTML.Window (RequestAnimationFrameId, Window)
+
+newtype DOMHighResTimestamp = DOMHighResTimestamp Number
+
+foreign import requestAnimationFrame :: (DOMHighResTimestamp -> Effect Unit) -> Window -> Effect RequestAnimationFrameId
```

Comment:
The Web.HTML.Window version of `requestAnimationFrame` does not expose the `DOMHighResTimestamp` argument to the callback. This is useful for animations that evolve over time. I'm not sure why it's omitted - perhaps worth upstreaming this change.